packages/global-styles/src/components/global-styles-variation-container/index.tsx

```diff
@@ -56,6 +57,15 @@ const GlobalStylesVariationContainer = ( {
 				visibility: width ? 'visible' : 'hidden',
 			} }
 			tabIndex={ -1 }
+			contentRef={ useRefEffect( ( bodyElement ) => {
```

Comment:
TIL 👀 
>https://developer.wordpress.org/block-editor/reference-guides/packages/packages-compose/#userefeffect
>Pass the returned ref callback as the component’s ref and merge multiple refs with useMergeRefs.

Passing the "effect" as a ref feels not intuitive for me, but it seems to work.